modules/social_features/social_content_report/src/ContentReportService.php

```diff
@@ -76,8 +76,9 @@ public function getReportFlagTypes() {
    * @param string $flag_id
    *   The flag ID.
    *
-   * @return array
-   *   A renderable array to be used in a #links array.
+   * @return array|bool
```

Comment:
Could it also work to just change our single `return FALSE;` into `return [];`? This way the return type is always the same (this makes using the function easier) and the empty array simply means 'nothing to render'.

Reply:
Agreed.